src/utils/PathManager.ts

```
@@ -11,95 +12,58 @@ export class PathManager {
     private isServerless: boolean
 
     private constructor() {
-        this.environment = process.env.ENVIRON || 'dev'
-        this.isServerless = process.env.SERVERLESS === 'true' || false
-        this.botUuid = null
-        this.secret = null
-        console.log('ENVIRON:', this.environment)
+        let global = GLOBAL.get()
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'global' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)